src/routes/github/create-branch/github-create-branch-options-get.ts

```diff
@@ -6,31 +6,33 @@ import { GitHubServerApp } from "~/src/models/github-server-app";
 export const GithubCreateBranchOptionsGet = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
 
 	const { jiraHost } = res.locals;
-	const { issue_key: key } = req.query;
+	const { issueKey } = req.query;
+	const { githubToken } = req.session;
 
```

Comment:
This is root level route not under `/github`.  We should not rely on githubtoken here

Reply:
Sweet we can chat about this, this afternoon, I need to determine if the user has a token at this point or not, if we redirect without the token it tried to oauth through the iframe and falls to pieces!!!

We might be able to send them to github-create-options and deal with the tokens at that point?


Or.... can we move this under /github route - we just can't have it using the github auth middlewares

Reply:
githubtoken ref removed